File: src/test/kotlin/jdbcTests/TestJdbc.java
package jdbcTests;

import com.rest.app.jdbc.SqlSelects;
import org.junit.Test;

import java.sql.SQLException;

public class TestJdbc {
    private final SqlSelects sqlSelects = new SqlSelects();

    @Test
    public void selectEmployeeById() throws SQLException {
        System.out.println("\n@Test selectEmp:");
        sqlSelects.selectEmployeeById(1);
        sqlSelects.selectEmployeeById(2);
        sqlSelects.selectEmployeeById(3);
    }

    @Test
    public void selectAllEmployee() throws SQLException {
        System.out.println("\n@Test selectAllEmp:");
        sqlSelects.selectAllStaff();
    }

    @Test
    public void selectEmployeeByName() throws SQLException {
        System.out.println("\n@Test selectAllEmp:");
        sqlSelects.selectEmployeeByName("Marcus Prince");
    }

    @Test
    public void createEmp() throws SQLException {
        System.out.println("\n@Test createEmp:");
        sqlSelects.addNewEmployee("Test User", "Tester", 89630165023L, java.sql.Date.valueOf("1990-01-01"));
    }

    @Test
    public void deleteEmp() throws SQLException {
        System.out.println("\n@Test deleteEmp:");
        sqlSelects.deleteEmployeeById(28);
    }
}

File: src/test/kotlin/jdbcTests/TestJDBC.kt
package jdbcTests

import com.rest.app.jdbc.SqlEmployeesSel
import org.junit.Test
import java.sql.Date
import java.sql.SQLException

class TestJDBC {

    private val sqlSelects = SqlEmployeesSel()

    @Test
    @Throws(SQLException::class)
    fun selectEmployeeById() {
        println("\n@Test selectEmp:")
        sqlSelects.selectEmployeeById(1)
        sqlSelects.selectEmployeeById(2)
        sqlSelects.selectEmployeeById(3)
    }

    @Test
    @Throws(SQLException::class)
    fun selectAllEmployee() {
        println("\n@Test selectAllEmp:")
        sqlSelects.selectAllStaff()
    }

    @Test
    @Throws(SQLException::class)
    fun selectEmployeeByName() {
        println("\n@Test selectAllEmp:")
        sqlSelects.selectEmployeeByName("Marcus Prince")
    }

    @Test
    @Throws(SQLException::class)
    fun createEmp() {
        println("\n@Test createEmp:")
        sqlSelects.addNewEmployee("Test User", "Tester", 89630165023L, Date.valueOf("1990-01-01"))
    }

    @Test
    @Throws(SQLException::class)
    fun deleteEmp() {
        println("\n@Test deleteEmp:")
        sqlSelects.deleteEmployeeById(28)
    }
}